NDTensors/src/lib/SparseArrayDOKs/src/sparsearraydok.jl

```diff
@@ -15,6 +15,10 @@ struct SparseArrayDOK{T,N,Zero} <: AbstractSparseArray{T,N}
 end
 
 # Constructors
+function SparseArrayDOK(data, dims::NTuple{N,Int}, zero) where {N}
+  return SparseArrayDOK{eltype(data),N,typeof(zero)}(data, dims, zero)
+end
```

Comment:
```suggestion
function SparseArrayDOK(data, dims::Tuple{Vararg{Int}}, zero)
  return SparseArrayDOK{eltype(data),length(dims),typeof(zero)}(data, dims, zero)
end
```
just to avoid the `where` syntax.